platform/source/minimal-printf/README.md

```diff
@@ -7,15 +7,15 @@ Prints directly to stdio/UART without using malloc. All flags and precision modi
 There is no error handling if a writing error occurs.
 
 Supports:
-* %d: signed integer [h, hh, (none), l, ll, z, j, t].
-* %i: signed integer [h, hh, (none), l, ll, z, j, t].
-* %u: unsigned integer [h, hh, (none), l, ll, z, j, t].
-* %x: unsigned integer [h, hh, (none), l, ll, z, j, t], printed as hexadecimal number (e.g., ff).
-* %X: unsigned integer [h, hh, (none), l, ll, z, j, t], printed as hexadecimal number (e.g., FF).
-* %f: floating point (disabled by default).
-* %F: floating point (disabled by default, treated as %f).
-* %g: floating point (disabled by default, treated as %f).
-* %G: floating point (disabled by default, treated as %f).
+* %d: signed integer [h, hh, (none), l, ll, z, j, t]. Width size (up to 48) and prepending zero modifiers are supported.
```

Comment:
It is not necessary to have a constraint on the width size, see suggestion in source file below.

Could you also edit the comment on line 6 `All flags and precision modifiers are ignored.`

Reply:
Done!